Doc/whatsnew/3.11.rst

```diff
@@ -169,6 +169,14 @@ Other Language Changes
   (Contributed by Serhiy Storchaka in :issue:`12022`.)
 
 
+Other CPython Implementation Changes
```

Comment:
@mdickinson Can you please take a look? Sorry, I am not a good English writer.

Reply:
@corona10 What you have looks fine to me!